src/client/automation/playback/press/key-press-simulator.js

```diff
@@ -118,7 +118,7 @@ export default class KeyPressSimulator {
 
         this._addKeyPropertyToEventOptions(eventOptions);
 
-        const raiseDefault = eventSimulator.keypress(activeElement, extend(eventOptions, modifiersState));
+        const raiseDefault = browserUtils.isAndroid || eventSimulator.keypress(activeElement, extend(eventOptions, modifiersState));
```

Comment:
I thought, may be extract `browserUtils.isAndroid` to separate function and cell it smth like `needRaiseKeyPress`

Reply:
I think you should not do it. In this case you will put in one method the condition and the whole action.
But by the name (`needRaiseKeyPress`), the method should contain only condition.